portal/src/main/webapp/WEB-INF/jsp/mutation_details.jsp

```diff
@@ -128,7 +128,16 @@
                         annotation: annotationCol.render
                     },
                     columnVisibility: {
-                        annotation: 'visible'
+                        annotation: 'visible',
+                        "tumorType": function() {
```

Comment:
@yichaoS are we sure that we don't want to keep the default behavior intact when `isVirtualStudy === false`?